cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -171,18 +182,19 @@ private Collection<AlterConfigOp> diff(int brokerId, String desired,
                     .findFirst();
 
             String op = d.get("op").asText();
+            boolean nodeIsController = nodeRef.controller();
```

Comment:
Same comment above applies here. I would just avoid having an extra variable and using `nodeRef.controller()` where needed.